intelmq/tests/bots/parsers/generic/custom_date_format.csv.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2022 gutsohnCERT
```

Comment:
```suggestion
SPDX-FileCopyrightText: 2022 CERT.at GmbH
```

Reply:
> 

I wonder why you want to change the license header from gutsohnCERT to CERT.at Gmbh. That does not make sense for me...

Reply:
"gutsohnCERT" was an employee of CERT.at